common/src/main/java/io/seata/common/util/IdWorker.java

```diff
@@ -144,12 +147,24 @@ public static IdWorker getInstance() {
         if (idWorker == null) {
             synchronized (IdWorker.class) {
                 if (idWorker == null) {
-                    init(ThreadLocalRandom.current().nextLong(1024));
+                    init(initWorkerId());
                 }
             }
         }
         return idWorker;
     }
+    public static long initWorkerId(){
+        InetAddress address;
+        try {
+            address = InetAddress.getLocalHost();
+        } catch (final UnknownHostException e) {
+            throw new IllegalStateException("Cannot get LocalHost InetAddress, please check your network!");
+        }
+        byte[] ipAddressByteArray = address.getAddress();
+        long workerId = (((ipAddressByteArray[ipAddressByteArray.length - 2] & 0B11) << Byte.SIZE) + (ipAddressByteArray[ipAddressByteArray.length - 1] & 0xFF));
+        Preconditions.checkArgument(workerId >= 0L && workerId < 1024L);
```

Comment:
if according to the current practice, will the id be greater than 1023? if yes, it is not advisable to simply throw an exception

Reply:
No more than 1023, because according to the algorithm, the maximum 1111111111 is 1023